tomster-player/app/templates/components/album-list.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="list-group sidebar">
+<div class="list-group {{if show-songs '' 'sidebar'}}">
```

Comment:
But if it's not rendered at all, doesn't that defeat the purpose of always showing the album list along with the album details?